### ISSUE TITLE
Change data_id?

### ISSUE THREAD

**Issue by vijaydewan** [man-group/dtale#601]
Hey Andrew,

Thanks again for all of the great work on this project!  I was wondering if there is an easy way to change the data_id once data is loaded?

e.g., --

startup("", data=test data, name="test", data_id=1)

[code to change data_id from 1 to 2]

I hope that makes sense.

Thanks!

**Comment by aschonfeld** [comment]
@vijaydewan there currently is not an easy way to do this.  I can work on adding a function `update_id(2)` to instance objects.  For example:

```
import pandas as pd
import dtale

d = dtale.show(pd.DataFrame([1,2,3,4,5])) # data_id -> 1
d.update_id(2) # data_id -> 2
```

I can also add a global function:

```
import pandas as pd
import dtale

d = dtale.show(pd.DataFrame([1,2,3,4,5]))
dtale.update_id(d._data_id, 2)
```

Will update once it's released.  Hope this is what you were picturing

**Comment by vijaydewan** [comment]
Thanks!

**Comment by aschonfeld** [comment]
@vijaydewan just released v2.2.0 with this update
